handlers/handlers.go

```diff
@@ -110,3 +115,72 @@ func GeographyRender(rend RenderClient) http.HandlerFunc {
 		return
 	}
 }
+
+//GeographyListpageRender ...
+func GeographyListpageRender(rend RenderClient) http.HandlerFunc {
```

Comment:
same here, no need for `Render` at the end of the handler name, and being pedantic but it possibly should be `GeographyListPage` rather than `GeographyListpage`